test/unit/services/test-get-overview.js

```diff
@@ -56,4 +64,18 @@ describe('services/get-overview', function () {
       done()
     })
   })
+
+  it('should return a results object with the correct overview details for a team', function (done) {
+    var teamName = 'team'
```

Comment:
Yea, thats a good idea going forward, will do that for all tests where we call this.